src/components/ActionButtons.jsx

```diff
@@ -0,0 +1,57 @@
+/* global _ */
+import React from 'react';
+import classnames from 'classnames';
+
+export default class ActionButtons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.elem = props.elem;
```

Comment:
You shouldn't do that in React. Each prop must be accessed via the `this.props` object, which is kept up-to-date when props are changed by the parent component. If you store and use the value like that without adding the code to update it, you'll render outdated prop values.